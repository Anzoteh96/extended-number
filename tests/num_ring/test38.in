-45583045073114331640337631218560398240706476048646842088479401488130051616295324551108324323621603375196546547163
463316259675092387833626523186647601910772666786258152161017475878470593083450040152143047528205833924642048418811
852925826145071549506848869486976623201576978767529852702956422958651672866579195172374431605085693628539716236667
