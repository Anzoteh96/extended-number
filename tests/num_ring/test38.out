Enter the number base b:
Enter two numbers n and m:
b is 45583045073114331640337631218560398240706476048646842088479401488130051616295324551108324323621603375196546547163
n mod b is 7485808943949071430250211001043619503707906299789731276223460997170076920496794641059804291989800172676582947181
m mod b is 32431014829013579980771507552889454868860409891886695110327196172310743773263353252424593779896832875001878387733
n + m is 39916823772962651411021718553933074372568316191676426386550657169480820693760147893484398071886633047678461334914
n - m is 20637839188049823089816334666714562875553972456549878254375666312989384763528765939743534835714570672871251106611
n * m is 22191153803013496088537078148605271624113496984791359766885182871687222927092674897327602583386298195354709620359
n ** m is 24245006616373060395206054878568724859302755337871183838507402067169388007058573003348093817497378870477230409612
